### PR TITLE
Update proofreading interface help

### PR DESCRIPTION
**Pull request by cpeel**
These are updates to the proofreading interface help to reflect
the changes made in the DPOD wiki. Updates courtesy of lhamilton.

### CODE REVIEW

**Comment by lhamilton1** [comment]
Approved

File: faq/prooffacehelp.php [hunk 1]
```diff
@@ -21,6 +21,10 @@
     echo "
     <center>
     <h2>Standard Proofreading Interface Help</h2>
+    <br>
+    <table>
+    <tr><td>Updated</td><td>05/24/2018</td><td>lhamilton</td></tr>
```
Comment (wf49670) [inline faq/prooffacehelp.php]: Suggestion: Use ISO standard date format, 2018-05-24

(Though, I see we use US format below, so maybe consistency is best.)

Comment (cpeel) [reply faq/prooffacehelp.php]: Yes, I'd prefer ISO-8601 as well but I think we should be consistent with the other dates in this file.

File: faq/prooffacehelp.php [hunk 2]
```diff
@@ -596,12 +637,13 @@ function echo_row( $name, $tooltip, $button_image_base, $accelerator )
 <P><A NAME="ikeys"> </A> <P><FONT SIZE="+1">Accelerator Keys (accesskeys)</FONT><P></TD></TR>
 <TR>
 <TD VALIGN="TOP" COLSPAN="2">
-Several of the commonly used interface buttons have been assigned an accesskey value.
+<p>These accelerator keys may not work for everyone.</p>
```
Comment (wf49670) [inline faq/prooffacehelp.php]: I would say: ... everyone, and the modifier key (shown as Alt below) may be different depending on your Operating System and browser.

File: faq/prooffacehelp.php [hunk 3]
```diff
@@ -697,6 +739,16 @@ function echo_row( $name, $tooltip, $button_image_base, $accelerator )
 <h4>Common Tags</h4>
 
 <p>
+You may sometimes find formatting already present in the text.
+Do not add or correct this formatting information; the formatters
+will do that later in the process. However, you can remove it if
+it interferes with your proofreading. The &lt;x&gt; button in the
```
Comment (wf49670) [inline faq/prooffacehelp.php]: It's not &lt;x&gt;
Rather, it is &lt;x&gt; but struckthrough, and should be shown that way.

File: faq/prooffacehelp.php [hunk 4]
```diff
@@ -752,20 +804,23 @@ function echo_row( $name, $tooltip, $button_image_base, $accelerator )
 </p>
 
 <h5>Shortcuts to insert common tags</h5>
+<p>
+    These shortcuts may not work for everyone.
```
Comment (wf49670) [inline faq/prooffacehelp.php]: ... for everyone, and the Alt may be a different key combination depending on your Operating System and browser.

File: faq/prooffacehelp.php [hunk 5]
```diff
@@ -752,20 +804,23 @@ function echo_row( $name, $tooltip, $button_image_base, $accelerator )
...
 <table border='1'>
   <tr><td>
-    Bold
+    Bold (Proofreaders do not add bolding codes)
   </td><td>
     Alt-b
   </td></tr>
   <tr><td>
-    Italics
+    Italics (Proofreaders do not add bolding codes)
```
Comment (wf49670) [inline faq/prooffacehelp.php]: bolding -> italic

File: faq/prooffacehelp.php [hunk 6]
```diff
@@ -752,20 +804,23 @@ function echo_row( $name, $tooltip, $button_image_base, $accelerator )
...
   </td><td>
     Alt-i
   </td></tr>
   <tr><td>
-    Small caps
+    Small caps (Proofreaders do not add bolding codes)
```
Comment (wf49670) [inline faq/prooffacehelp.php]: bolding -> Small caps

File: faq/prooffacehelp.php (hunk 2 above)
Comment (cpeel) [reply faq/prooffacehelp.php]: Please coordinate with @lhamilton1 on the wording of this, whatever you both decide is fine with me and needs to be updated on the wiki as well.

File: faq/prooffacehelp.php (hunk 4 above)
Comment (cpeel) [reply faq/prooffacehelp.php]: Ditto, please coordinate with @lhamilton1.

File: faq/prooffacehelp.php (hunk 5 above)
Comment (cpeel) [reply faq/prooffacehelp.php]: Fixed, thanks.

File: faq/prooffacehelp.php (hunk 6 above)
Comment (cpeel) [reply faq/prooffacehelp.php]: Fixed, thanks.

File: faq/prooffacehelp.php (hunk 3 above)
Comment (cpeel) [reply faq/prooffacehelp.php]: Good catch, fixed.

**Comment by lhamilton1** [comment]
I'm OK with the changes, Walt suggest. Once it's in Test I'll take a close look to double-check but liked the added phrasing.

    On Monday, June 4, 2018, 11:30:42 AM EDT, Casey Peel <notifications@github.com> wrote:  
 
 
@cpeel commented on this pull request.

In faq/prooffacehelp.php:
> @@ -596,12 +637,13 @@ function echo_row( $name, $tooltip, $button_image_base, $accelerator )
 <P><A NAME="ikeys"> </A> <P><FONT SIZE="+1">Accelerator Keys (accesskeys)</FONT><P></TD></TR>
 <TR>
 <TD VALIGN="TOP" COLSPAN="2">
-Several of the commonly used interface buttons have been assigned an accesskey value.
+<p>These accelerator keys may not work for everyone.</p>

Please coordinate with @lhamilton1 on the wording of this, whatever you both decide is fine with me and needs to be updated on the wiki as well.

File: faq/prooffacehelp.php (hunk 2 above)
Comment (cpeel) [reply faq/prooffacehelp.php]: Updated.

File: faq/prooffacehelp.php (hunk 4 above)
Comment (cpeel) [reply faq/prooffacehelp.php]: Updated.

File: faq/prooffacehelp.php [hunk 7]
```diff
@@ -569,6 +604,7 @@ function echo_row( $name, $tooltip, $button_image_base, $accelerator )
 echo_row( 'Change Interface Layout', 'Change Interface Layout', 'bt4+bt5', '6' );
 // echo_row( 'Check for Common Errors', 'Check for Common Errors', 'bt6', '' );
 echo_row( 'Run WordCheck', 'Run WordCheck', 'bt16', '' );
+echo_row( 'Preview', 'Format preview', 'bt20', '' );
```
[the remark this answers is not in the document]
Comment (cpeel) [reply faq/prooffacehelp.php]: No. The left one is what will be displayed on the Standard Interface button, and selects the text, and the right is the tooltip.

File: faq/prooffacehelp.php [hunk 8]
```diff
@@ -569,6 +604,7 @@ function echo_row( $name, $tooltip, $button_image_base, $accelerator )
 echo_row( 'Change Interface Layout', 'Change Interface Layout', 'bt4+bt5', '6' );
```
[the remark this answers is not in the document]
Comment (cpeel) [reply faq/prooffacehelp.php]: There was some invalid HTML (not escaping tooltips in HTML attributes) but that wasn't the problem. The problem is `lang_dir()` cruft that needs to be removed. I'll follow up with a separate PR for that.

**Pull request merged**